ateam_radio_bridge/src/conversion.hpp

```diff
@@ -24,15 +24,17 @@
 
 #include <ateam_msgs/msg/robot_feedback.hpp>
 #include <ateam_msgs/msg/robot_motion_command.hpp>
+#include <ateam_msgs/msg/robot_motion_feedback.hpp>
 #include <basic_control.h>
 #include <basic_telemetry.h>
+#include <control_debug_telemetry.h>
 #include <hello_data.h>
 
 namespace ateam_radio_bridge
 {
 
-ateam_msgs::msg::RobotFeedback Convert(const BasicTelemetry & basic_telemetry);
-
+ateam_msgs::msg::RobotFeedback ConvertBasicTelemetry(const BasicTelemetry & basic_telemetry);
+ateam_msgs::msg::RobotMotionFeedback ConvertControlDebugTelemetry(const ControlDebugTelemetry & control_debug_telemetry);
```

Comment:
nit: most of our conversion helpers in other packages stick to a consistent name and let overload deduction pick the right one for us.

So this section could end up looking like this:
```suggestion
ateam_msgs::msg::RobotFeedback Convert(const BasicTelemetry & basic_telemetry);
ateam_msgs::msg::RobotMotionFeedback Convert(const ControlDebugTelemetry & control_debug_telemetry);
```